Examples/Io/NuclearInteractions/include/ActsExamples/Io/NuclearInteractions/RootNuclearInteractionParametersWriter.hpp

```diff
@@ -0,0 +1,80 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020-2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "ActsExamples/Framework/WriterT.hpp"
+#include "ActsExamples/Io/NuclearInteractions/detail/NuclearInteractionParametrisation.hpp"
+
+#include <mutex>
+#include <vector>
+
+namespace ActsExamples {
+
+struct ExtractedSimulationProcess;
```

Comment:
You could just include the `ActsExamples/EventData/ExtractedSimulationProcess.hpp` instead of the forward declaration. `ExtractedSimulationProcessContainer` is also used below.

Reply:
changed